evadb/configuration/bootstrap_environment.py

```diff
@@ -46,7 +48,27 @@ def get_base_config(evadb_installation_dir: Path) -> Path:
 
 
 def get_default_db_uri(evadb_dir: Path):
-    return f"sqlite:///{evadb_dir.resolve()}/{DB_DEFAULT_NAME}"
+    """
+    Get the default database uri.
+
+    Arguments:
+        evadb_dir: path to evadb database directory
+    """
+    config_obj = parse_config_yml()
+    if config_obj["experimental"]["use_postgres_backend"] is False:
+        return f"sqlite:///{evadb_dir.resolve()}/{DB_DEFAULT_NAME}"
+    else:
+        # Custom Postgres server arguments need to be set in the evadb.yml file
+        pg_host = config_obj["postgres"]["pg_host"]
+        pg_port = config_obj["postgres"]["pg_port"]
+        pg_user = config_obj["postgres"]["pg_user"]
+        pg_password = config_obj["postgres"]["pg_password"]
+
+        if not pg_password:
+            return f"postgresql://{pg_user}@{pg_host}:{pg_port}/{PG_DB_DEFAULT_NAME}"
```

Comment:
What do you think if we ask the user to provide this string directly instead of 4 parameters?

Reply:
Right now, `catalog_database_uri` in `config.yml`  is populated with `sqlite` string. I suggest we use it for postgres. Users can change it to Postgres. 
Let me know if I'm missing some complexity with this approach. Thanks!

Reply:
Main concern was if users set 'use_postgres_backend' to true without setting the catalog_database_uri field, they will get an authentication issue since we don't have a default pg uri. However your suggested approach is much simpler, I have made the changes and added comments to ensure the field is set.